### PR TITLE
Use manipulator methods

### PR DESCRIPTION
**Pull request by rohit2sharma95**
Use manipulator methods everywhere to make consistency in the codebase

### CODE REVIEW

File: js/src/carousel.js [hunk 1]
```diff
@@ -418,7 +418,7 @@ class Carousel extends BaseComponent {
       return
     }
 
-    const elementInterval = Number.parseInt(element.getAttribute('data-bs-interval'), 10)
+    const elementInterval = Manipulator.getDataAttribute(element, 'interval')
```
Comment (XhmikosR) [inline js/src/carousel.js:421]: `Number.parentInt` should probably stay, right?

Comment (rohit2sharma95) [reply js/src/carousel.js:421]: The data is already being normalized in manipular:
```js
if (val === Number(val).toString()) {
  return Number(val)
}
```

But that does not parse the number in integer though :thinking:

Comment (rohit2sharma95) [reply js/src/carousel.js]: Updated :+1:

**Comment by XhmikosR** [comment]
I'm not sure this is worth doing.

Let's revisit later if needed.

**Pull request closed without merging**